README.md

```
@@ -82,6 +82,7 @@ uvx hooks-mcp
 
 Optional command line arguments include:
  - `--working-directory`/`-wd`: Typically the path to your project root. Set if not running from project root.
+ - `--http-streaming-port`: Run the server with HTTP streaming on the specified port instead of stdio. The server will listen on `http://localhost:<port>/mcp`.
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 1

(MD007, ul-indent)